klarna-on-demand/src/main/java/com/klarna/ondemand/Context.java

```diff
@@ -24,4 +26,29 @@ protected static String getApiKey(){
         }
         return apiKey;
     }
+
+    /**
+     * Sets the button color for Klarna views in this application.
+     * @param buttonColor Merchant's public API key for this application.
+     */
+    public static void setButtonColor(Integer buttonColor) { Context.buttonColor = buttonColor; }
```

Comment:
Colors are treated as resources - integers are used as the ID for that color from the resources mapping. 
http://developer.android.com/reference/android/content/res/Resources.html#getColor(int)